src/Routes/DeviceDetail/DeviceDetail.js

```diff
@@ -16,7 +16,7 @@ import {
   InventoryDetailHead,
   DetailWrapper,
 } from '@redhat-cloud-services/frontend-components/Inventory';
-import { useParams } from 'react-router-dom';
+import { useParams, useLocation, useHistory } from 'react-router-dom';
```

Comment:
Combine with the `Link` import on line 2. Also, `useLocation` isn't needed (see comments below).